doc/user-guide/user-guide.rst

```diff
@@ -0,0 +1,103 @@
+.. _User_Guide:
+
+#####################
+`ts_imsim` User Guide
+#####################
+
+The `ts_imsim` closed loop is designed to run as an executable through the command line as `imgClosedLoop.py`.
+
+.. _Installing_ts_imsim:
+
+Installing `ts_imsim`
+=====================
+
+Needed Packages
+---------------
+
+- `ts_wep <https://github.com/lsst-ts/ts_wep>`_
+- `ts_ofc <https://github.com/lsst-ts/ts_ofc>`_
+- `imSim <https://github.com/LSSTDESC/imSim>`_
+- `astroplan <https://github.com/astropy/astroplan>`_
+- `black <https://github.com/psf/black>`_ (optional)
+- `documenteer <https://github.com/lsst-sqre/documenteer>`_ (optional)
+- `plantuml <http://plantuml.com>`_ (optional)
+- `sphinxcontrib-plantuml <https://pypi.org/project/sphinxcontrib-plantuml/>`_ (optional)
+
+Install ts_imsim
+-----------------------------------------
+
+1. Install and set up required packages from list above using `eups`.
+
+2. Clone the `ts_imsim` package.
+
+3. Setup the imSim environment by `eups`:
+
+```bash
+cd $ts_imsim_directory
+setup -k -r .
+scons
+```
```

Comment:
In *rst*, the syntax is the following (note there is the indentation that the GitHub Preview does not show it.):

"""
.. code:: python
    cd $ts_imsim_directory
    setup -k -r .
    scons
"""

Another example:

<img width="595" alt="Screenshot 2023-11-07 at 9 18 17 PM" src="https://github.com/lsst-ts/ts_imsim/assets/11474902/33b69a42-6258-4d59-a92e-a6c86d930709">

See "https://sublime-and-sphinx-guide.readthedocs.io/en/latest/code_blocks.html" or others.

Otherwise, it looks like this:
![Uploading Screenshot 2023-11-07 at 9.17.16 PM.png…]()